src/parser/parser_support.c

```diff
@@ -153,7 +153,8 @@ static PT_NODE *pt_make_query_user_groups (PARSER_CONTEXT * parser, const char *
 static void pt_help_show_create_table (PARSER_CONTEXT * parser, PT_NODE * table_name, string_buffer & strbuf);
 static int pt_get_query_limit_from_orderby_for (PARSER_CONTEXT * parser, PT_NODE * orderby_for, DB_VALUE * upper_limit,
 						bool * has_limit);
-static int pt_get_query_limit_from_limit (PARSER_CONTEXT * parser, PT_NODE * limit, DB_VALUE * limit_val);
+static int pt_get_query_limit_from_limit (PARSER_CONTEXT * parser, PT_NODE * limit, DB_VALUE * limit_val,
```

Comment:
I suggest to rename to **pt_get_query_limit_from_limit_clause** for readability.